cookbooks/cdo-varnish/libraries/http_cache.rb

```diff
@@ -7,6 +7,8 @@ class HttpCache
 
   # Language header and cookie are needed to separately cache language-specific pages.
   LANGUAGE_HEADER = %w(Accept-Language).freeze
+  COUNTRY_HEADER = %w(CloudFront-Viewer-Country).freeze
+  WHITELISTED_HEADERS = LANGUAGE_HEADER + COUNTRY_HEADER
```

Comment:
We've been trying to avoid this phrase.  "Accept list" or something else instead?

Reply:
I'm leaving this for now for consistency with the rest of the existing file and with the [CloudFront API](https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/header-caching.html) terminology, but will follow up on whether we have a need to adopt/clarify naming guidelines moving forward.